lib/new_relic/agent/instrumentation/grape.rb

```diff
@@ -9,7 +9,7 @@
 DependencyDetection.defer do
   # Why not just :grape? newrelic-grape used that name already, and while we're
   # not shipping yet, overloading the name interferes with the plugin.
-  named :grape_instrumentation
+  @name = :grape_instrumentation
```

Comment:
Why did we make this change to select instrumentation and not others?

Reply:
This was related to the [configuration options we removed](https://github.com/newrelic/newrelic-ruby-agent/pull/1782). Several of these are called `something_instrumentation`, but our configuration options are `instrumentation.something`. We have tests that expect configuration options called `instrumentation.something_instrumentation` when `named` is used, so we had to switch it to just set the name attribute directed.